scripts/flattener_mods/constants.py

```diff
@@ -190,3 +193,86 @@
 	'GENCODE 20': 'v76',
 	'GENCODE 19': 'v74',
 }
+
+SAMPLE_COLLECTION_MAP = {
+	'percutaneous biopsy': 'biopsy',
+	'open biopsy': 'biopsy',
+	'resection': 'surgical resection',
+	'dissection': 'surgical resection',
+	'swab': 'brush',
+	'bronchoalveolar lavage': 'bodily fluid',
+	'aspiration': 'biopsy',
+	'density centrifugation': 'other',
+	'enzymatic digestion': 'other',
+	'cryosection': 'other'
+}
+
+SAMPLE_PRESERVATION_MAP = {
+	'cyropreservation': 'frozen at -80C',
```

Comment:
Just randomly happened to catch this typo trying LATDF519CUZ; didn't notice with careful reading for misspellings. `cryopreservation` remained in the final object instead of remapping to `frozen at -80C`

Reply:
Thank you for catching that! :) I've pushed a commit.